src/Controllers/ImageController.php

```diff
@@ -72,6 +72,46 @@ public function __invoke(Request $request, int $store, string $size, string $pla
         return $this->storage()->response($resizedPath);
     }
 
+    public function getResizedPath(string $size, string $file, string $webp)
+    {
+        foreach (config('imageresizer.external') as $placeholder => $url) {
+            if (Str::startsWith($file, $placeholder)) {
+                $file = Str::replaceFirst($placeholder, '', $file);
+                $placeholderUrl = $url;
+                break;
+            }
+        }
+
+        $placeholder = isset($placeholderUrl)
+            ? $placeholder
+            : 'local';
+
+        return config('rapidez.store').'/resizes/'.$placeholder.'/'.$size.'/'.$file.$webp;
+    }
+
+    public function redirectFromSku(Request $request, string $size, string $file)
+    {
+        $webp = str_ends_with($file, '.webp') ? '.webp' : '';
+        $baseFile = $webp ? str_replace_last('.webp', '', $file) : $file;
```

Comment:
This can be prevented by adding it as an optional parameter to the url
like done with the regular route:
https://github.com/rapidez/image-resizer/blob/377cb93cb6277e3fc63e37cbf890fc871fdd2e5c/src/ImageResizerServiceProvider.php#L19
https://github.com/rapidez/image-resizer/blob/377cb93cb6277e3fc63e37cbf890fc871fdd2e5c/src/ImageResizerServiceProvider.php#L20

Reply:
This is something I originally tried, but the formatting became problematic due to the fact that a SKU can contain a `.`. I wasn't able to find a clean way around that.

Reply:
Yes, the matching will not be a copy paste of the original regex as that has been made to match
`imagename.jpg.webp`
skus don't have a file extension thus this exact regex would never work well with it